pkg/kubelet/kubelet.go

```diff
@@ -775,6 +780,14 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 
 	tokenManager := token.NewManager(kubeDeps.KubeClient)
 
+	// Initialize csiDriverLister before calling NewInitializedVolumePluginMgr
+	// because csiDriverLister is needed in csi_plugin.go
```

Comment:
it shouldn't be needed in init though? so I think its safe to initialize later

Reply:
Moved it to NewInitializedVolumePluginMgr.